### PR TITLE
Use deviator for computing equivalent strain

### PR DESCRIPTION
**Pull request by psavery**

### CODE REVIEW

**Review by joelvbernier — APPROVED** [review]

**Review by joelvbernier — CHANGES_REQUESTED** [review]
var name

File: hexrd/ui/indexing/fit_grains_results_dialog.py [hunk 1]
```diff
@@ -61,8 +61,9 @@ def __init__(self, data, material=None, parent=None):
         # Add column for equivalent strain
         eqv_strain = np.zeros(self.num_grains)
         for i, grain in enumerate(self.data):
-            emat = vecMVToSymm(grain[15:21], scale=False)
-            eqv_strain[i] = 2 * np.sqrt(np.sum(emat**2)) / 3
+            epsilon = vecMVToSymm(grain[15:21], scale=False)
+            deviator = epsilon - (1/3) * np.trace(epsilon) * np.identity(3)
+            eqv_strain[i] = 2 * np.sqrt(np.sum(epsilon**2)) / 3
```
Comment (joelvbernier) [inline hexrd/ui/indexing/fit_grains_results_dialog.py:66]: eqv_strain[i] = 2 * np.sqrt(np.sum(deviator**2)) / 3

**Review by joelvbernier — CHANGES_REQUESTED** [review]

File: hexrd/ui/indexing/fit_grains_results_dialog.py (hunk 1 above)
Comment (psavery) [reply hexrd/ui/indexing/fit_grains_results_dialog.py]: Nice catch Joel! It should be fixed now.

**Review by cjh1 — APPROVED** [review]

**Comment by psavery** [comment]
@joelvbernier Does it look good? Looks like you need to approve it...

**Review by joelvbernier — APPROVED** [review]

**Pull request merged**